content/en/docs/File Format/implementationstatus.md

```diff
@@ -0,0 +1,178 @@
+| ZSTD                                      |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| LZ4_RAW                                   |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+
+### Other format level features
+
++-------------------------------------------+-------+--------+--------+-------+-------+
+|                                           | C++   | Python | Java   | Go    | Rust  |
+|                                           |       |        |        |       |       |
++===========================================+=======+========+========+=======+=======+
+| xxHash Bloom filters                      |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| bloom filter length                       |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Statistics min_value, max_value           |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Column index                              |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Offset index                              |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Modular encryption                        |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Page CRC32 checksum                       |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Modular encryption                        |       |        |        |       |       |
++-------------------------------------------+-------+--------+--------+-------+-------+
+
+High level data API-s for parquet feature usage
+===============================================
+
++----------------------------------------------+-------+--------+--------+-------+-------+
+| Format                                       | C++   | Python | Java   | Go    | Rust  |
+|                                              |       |        |        |       |       |
++==============================================+=======+========+========+=======+=======+
+| Hive-style partitioning                      |       |        |        |       |       |
++----------------------------------------------+-------+--------+--------+-------+-------+
+| Partition pruning on the partition column    |       |        |        |       |       |
```

Comment:
I think these two features could probably be left out.  I can see how the others are high-level API features but they are still very much "exposing parquet capabilities".  I think hive-style partitioning is completely unrelated to parquet the format though.

Reply:
The number of systems which support parquet, but not hive style partitioning is very limited. As in the intro I've explained my goal was to emphasize the different capabilities and levels of abstraction of the different implenetations. If we are strict about what's in the parquet format and ignore all the high level capabilities eg the pyarrow matches about 0% of the format as it can't access, manipulate or create parquet without the intermediate arrow format (unlike in Java and others)

Reply:
(I don't feel very strongly about this, but also we already have doxygen/sphinx and other API docs which are really good, so... yeah, maybe it's better to have it in a blogpost comparing / demoing the features)

Reply:
Since this is in a "high level APIs" section I think it's reasonable to include, and informative for the user.

Reply:
We can rename it to “integrations” to express this better. Would that be more suitable?